src/views/debug/view_debug_misc.cpp

```diff
@@ -10,13 +10,16 @@
 #include "util/ped.hpp"
 #include "util/system.hpp"
 #include "view_debug.hpp"
+#include "gta/color.hpp"
 
 #include "hooking.hpp"
 
 namespace big
 {
 	void debug::misc()
 	{
+		rage::rgbaColor a;
```

Comment:
Unused local var

Reply:
true, used them to test the classes, its not used anywhere maybe delete since there is also ImU32 or just float[4]?